practice/pyramid/site.css

```diff
@@ -0,0 +1,38 @@
+
+.row {
+  display: flex;
+  border: 1px solid black;
+  margin: 20px;
+  height: 100px;
+  justify-content: space-between;
+}
+
+.first-row-item {
+  flex: 1;
+  margin: 10px;
+  border: 1px solid black;
+}
+
+.second-row-item1 {
+  flex: 4;
+  margin: 10px;
+  border: 1px solid black;
+}
+
+.second-row-item2 {
+  flex: 1;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 0;
+  margin-right: 10px;
+  border: 1px solid black;
+}
+
+.third-row-item {
+  flex: 1;
+  margin: 10px;
+  flex-basis: 90px;
+  flex-grow: 0;
+  flex-shrink: 0;
```

Comment:
In the case of a fixed with item, it is easier to directly specify `width` than `flex-basis` and `flex: 0`. You aren't required to set `flex` on all children of a `display: flex` container.